camelot/view/validator.py

```diff
@@ -45,6 +45,21 @@
 
 data_validity = collections.namedtuple('data_validity', ['valid', 'value', 'formatted_value', 'error_msg', 'info'])
 
+@dataclass
+class ValidatorState(DataclassSerializable):
+
```

Comment:
Deze gaat dus de data_validity namedtuple vervangen nadat de national identifier en zipcode validators zijn omgezet in de volgende PR.